README.md

```diff
@@ -1,6 +1,6 @@
-# MaterialsZone Bokeh Package
+## MaterialsZone Bokeh Package
 
-## Installation
+### 1. Installation
 To install the latest version of this package in your current environment do:
 ```bash
 pip install git+https://github.com/materialscloud/mz-bokeh-package.git@master
```

Comment:
@ofir-frd - please replace in this line and in line 11 "@master" with the version number (in other words, the tag).